filesystem/src/main/java/com/nytimes/android/external/fs/SourcePersister.java

```diff
@@ -54,4 +51,9 @@ static String pathForBarcode(@Nonnull BarCode barCode) {
         return sourceFileWriter.write(barCode, data);
     }
 
+    @Nonnull
+    static String pathForBarcode(@Nonnull BarCode barCode) {
+        return barCode.toString();
```

Comment:
It's not my change. For some reason this method is showing up a few lines below. I have reverted it back to how it is in develop branch